scargo/scargo_src/sc_debug.py

```diff
@@ -28,31 +28,37 @@ def __init__(self, config: Config, bin_path: Optional[Path]):
             )
             sys.exit(1)
 
-        self._bin_path = bin_path or self._get_bin_path(config.project.bin_name)
+        bin_name = config.project.bin_name
```

Comment:
Why change this? bin_name is Optional[str]

Reply:
Because it's used as argument to `self._get_bin_path` which takes `str`. But I see it's only relevant when `bin_path` is empty, so I'll change it a little.

Reply:
Sure